test/pytorch_test_base.py

```diff
@@ -444,7 +444,9 @@ def skipped_test(self, *args, reason=reason, **kwargs):
                   TORCH_TEST_PRECIIONS,
                   [dtype_test_name, test_name, test.__name__],
                   DEFAULT_FLOATING_PRECISION)
-              test.precision_overrides[dtype] = floating_precision
+              if dtype not in test.precision_overrides or test.precision_overrides[
```

Comment:
Maybe:
```python
test.precision_overrides[dtype] = max(test.precision_overrides[dtype], floating_precision)
```
Reads a bit better for me.

Reply:
Doing `max(test.precision_overrides[dtype], floating_precision)` might cause a keyError right? I don't think every test sets precision_overrides for all dtypes.

Reply:
Oh, then yes, leave it as is.